src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -92,7 +93,7 @@ public ScriptProjectSystem(OmniSharpWorkspace workspace, IOmniSharpEnvironment e
 
         public string Key => "Script";
         public string Language => LanguageNames.CSharp;
-        public IEnumerable<string> Extensions => new[] { ".csx" };
+        public IEnumerable<string> Extensions => new[] { CsxExtension };
```

Comment:
FYI: The `=>` syntax can cause an array to be created each time the property is accessed (though the compiler might have cached it somewhere due to it containing only constant expressions). I think that might be the case for this one.